src/libstd/num/f32.rs

```diff
@@ -75,16 +75,86 @@ mod cmath {
 
 #[stable(feature = "rust1", since = "1.0.0")]
 impl Float for f32 {
+    /// Return `NaN`
```

Comment:
I think this should be ``Returns `NaN`.`` (with an s and a ‘.’)—the indicative style is recommended over the imperative.

(Same thing for the others below.)